frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -0,0 +1,77 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import * as loginView from '@console/internal-integration-tests/views/login.view';
+import { withResource, removeLeakableResource } from '@console/shared/src/test-utils/utils';
+import { createProject } from './utils/utils';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { VM_ACTION } from './utils/consts';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const {
+  BRIDGE_HTPASSWD_IDP = 'test',
+  BRIDGE_HTPASSWD_USERNAME = 'test',
+  BRIDGE_HTPASSWD_PASSWORD = 'test',
+} = process.env;
+const configName = ProvisionConfigName.URL;
+const leakedResources = new Set<string>();
+const provisionConfigs = getProvisionConfigs(testName);
+
+const provisionConfig = provisionConfigs.get(configName);
+provisionConfig.networkResources = [];
+provisionConfig.storageResources = [];
+
+describe('Kubevirt non-admin VM Creation', async () => {
+  beforeAll(async () => {
+    await browser.get(appHost);
+    await browser.sleep(3000); // Wait long enough for the login redirect to complete
+  });
+  describe('Non-Admin Flow Test', async () => {
+    beforeAll(() => {
+      // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
```

Comment:
This is not necessary, you can pass custom timeout to `it` method as 3rd prameter:
```
it('spec name', async () => {
  ...
},
CUSTOM_TIMEOUT); 
```
Same way to before/after All/Each methods.